sdk/storage/Azure.Storage.Common/src/Shared/AesGcm/AesGcmWindows.cs

```diff
@@ -0,0 +1,143 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+using System;
+using System.Security.Cryptography;
+
+namespace Azure.Storage
+{
+    /// <summary>
+    /// Taken from
+    /// https://github.com/dotnet/runtime/blob/main/src/libraries/System.Security.Cryptography/src/System/Security/Cryptography/AeadCommon.Windows.cs
```

Comment:
I don't see the `AesGcmWindows` class at this link.  Also, can we use a specific SHA (you can easily get this by hitting the Y key on github's site to update the browser's path from `main`)?  For example, it gives me something more like https://github.com/dotnet/runtime/blob/89cee6e3fbd21ad79be00a735a017f0c932c1501/src/libraries/System.Security.Cryptography/src/System/Security/Cryptography/AeadCommon.Windows.cs for this path.

Reply:
Also, does this comment apply to the class or the entire file?  We might consider moving it to the top of the file if it applies to everything.